fecgov/fec-cms
# Fix guidance search on load

## Summary

_I noticed that the guidance page was not loading on load. It had this error below. This is because I did not make sure results appear before doing a count, so none was returned. This patch ensures that total_count always has a value._

total_count = results['meta']['count'] + results['best_bets']['count']
TypeError: 'NoneType' object is not subscriptable

## Impacted areas of the application

List general components of the application that this PR will affect:

-  Guidance search

## Related PRs

List related PRs against other branches:

branch | PR
------ | ------
feature/3662-add-best-bets-to-count | [https://github.com/fecgov/fec-cms/pull/3663]()

## How to test

- Pull down this PR
- `export SEARCH_GOV_POLICY_GUIDANCE_KEY` in your environment vars. You can grab this API access key by logging into search.gov and accessing the fec_content_s3 site.
- `export FEC_FEATURE_GUIDANCE_SEARCH = true`
- Go to http://localhost:8000/legal-resources/policy-and-other-guidance/guidance-documents/ and make sure it loads
- Try a search that includes best bets search like `citizens united`. The total count should add best bets and results count to give an accurate count 
____
